app/models/rubric_criterion.rb

```diff
@@ -1,54 +1,49 @@
 class RubricCriterion < Criterion
-  before_validation :scale_marks_if_max_mark_changed
   before_save :round_max_mark
 
   validates :levels, presence: true
 
   DEFAULT_MAX_MARK = 4
   # Checks whether the passed in param's level_attributes have unique name and marks.
   # Skips the uniqueness validation if true.
-  def update_levels(levels_attributes)
+  def update_levels(levels_attributes, scale = 1.0)
```

Comment:
Previously we were looking at changes to `max_mark` using `self.changes` (which I figured was a callback thing, not sure if that's available/how that works here) - to avoid some weird logic with the same I just made it a parameter the caller can very easily calculate.